test/box-luatest/gh_4693_formats_for_standalone_tuples_test.lua

```diff
@@ -37,6 +37,18 @@ g.test_box_tuple_format_new = function()
     end)
 end
 
+-- Checks that improper use of `box.internal.tuple_format.new` doesn't crash.
+g.test_box_internal_tuple_format_new = function(cg)
+    cg.server:exec(function()
+        t.assert_error_msg_content_equals(
+            "format should be a table",
+            function() box.internal.tuple_format.new() end)
```

Comment:
We don't need wrappers with error assertions, we can just pass function and its arguments.

Reply:
Removed.